src/Text/Pandoc/Class.hs

```diff
@@ -573,9 +573,9 @@ downloadOrRead s = do
     (Nothing, s') ->
        case parseURI s' of  -- requires absolute URI
             -- We don't want to treat C:/ as a scheme:
```

Comment:
That comment should probably be moved, too.

Reply:
of course, thanks!